contract-ui/tabs/overview/helpers/getGuidesAndTemplates.ts

```diff
@@ -0,0 +1,386 @@
+    url: "https://blog.thirdweb.com/guides/mint-nft-unique-code/",
+  },
+  airdropEditionNftsToHoldersOfAnNftCollection: {
+    title: "Airdrop Edition NFTs to Holders of an NFT Collection",
+    url: "https://blog.thirdweb.com/guides/airdrop-edition-nfts-to-holders-of-an-nft-collection/",
+  },
+  createAnNftGalleryUsingThirdwebAndNextJs: {
+    title: "Create an NFT Gallery using thirdweb and Next.js",
+    url: "https://blog.thirdweb.com/guides/create-an-nft-gallery-using-thirdweb-and-next-js/",
+  },
+  nftGatedWebsite: {
+    title: "Create An NFT Gated Website",
+    url: "https://blog.thirdweb.com/guides/nft-gated-website/",
+  },
+  allowGithubContributorsToMintAnNft: {
+    title: "Allow GitHub contributors to mint an NFT",
+    url: "https://blog.thirdweb.com/guides/allow-github-contributors-to-mint-an-nft/",
+  },
+  airdropFreeToOwnNftsForAWeb3Game: {
+    title: "Airdrop Free-To-Own NFTs For Your Web3 Game",
+    url: "https://blog.thirdweb.com/guides/airdrop-free-to-own-nfts-for-a-web3-game/",
+  },
+  createAnNftLootbox: {
+    title: "Create NFT Loot-Boxes Using the Pack Contract",
+    url: "https://blog.thirdweb.com/guides/create-an-nft-lootbox/",
+  },
+  signatureDrop: {
+    title: "Create an ERC721A NFT Drop with Signature-Based Minting",
+    url: "https://blog.thirdweb.com/guides/signature-drop/",
+  },
+  nftMarketplaceWithTypescriptNext: {
+    title: "Create Your Own NFT Marketplace with TypeScript and Next.js",
+    url: "https://blog.thirdweb.com/guides/nft-marketplace-with-typescript-next/",
+  },
+  auctionButtonReact: {
+    title: "Build An Auction Button For Your NFT Marketplace",
+    url: "https://blog.thirdweb.com/guides/auction-button-react/",
+  },
+  buildTreasuryAndGovernanceForYourDao: {
+    title: "Build a DAO With a Treasury and a Governance Token",
+    url: "https://blog.thirdweb.com/guides/build-treasury-and-governance-for-your-dao/",
+  },
+};
+
+const GUIDES_FOR_TYPE: { [key: string]: Item[] } = {
+  pack: [ALL_GUIDES.createAnNftLootbox],
+  signatureDrop: [ALL_GUIDES.signatureDrop],
+  marketplace: [
+    ALL_GUIDES.nftMarketplaceWithTypescriptNext,
+    ALL_GUIDES.auctionButtonReact,
+  ],
+  vote: [ALL_GUIDES.buildTreasuryAndGovernanceForYourDao],
+};
+
+const GUIDES_FOR_FEATURE: { [key: string]: Item[] } = {
+  ERC20ClaimConditionsV1: [
+    ALL_GUIDES.claimErc20TokenNextjs,
+    ALL_GUIDES.sellYourInGameCurrencyForCryptoInUnity,
+  ],
+  ERC20ClaimConditionsV2: [
+    ALL_GUIDES.claimErc20TokenNextjs,
+    ALL_GUIDES.sellYourInGameCurrencyForCryptoInUnity,
+  ],
+  ERC20ClaimPhasesV2: [
+    ALL_GUIDES.claimErc20TokenNextjs,
+    ALL_GUIDES.sellYourInGameCurrencyForCryptoInUnity,
+  ],
+  ERC20ClaimPhasesV1: [
+    ALL_GUIDES.claimErc20TokenNextjs,
+    ALL_GUIDES.sellYourInGameCurrencyForCryptoInUnity,
+  ],
+  ERC20Burnable: [],
+  ERC20SignatureMintable: [],
+  ERC20BatchMintable: [],
+  ERC20Mintable: [],
+  ERC20: [
+    ALL_GUIDES.tokenButtonReact,
```

Comment:
this should be only for ERC20 with claim conditions / claim phases not for all ERC20s